app/authenticated/cluster/index/route.js

```diff
@@ -6,11 +6,12 @@ export default Route.extend({
   scope: service(),
 
   model: function () {
-    return this.get('globalStore').findAll('machine', { filter: { clusterId: this.get('scope.currentCluster.id') } }).then((nodes) => {
+    return this.get('globalStore').findAll('machine').then((nodes) => {
       const cluster = this.get('scope').currentCluster;
+
       return {
         displayName: cluster.name,
-        nodes,
+        nodes: nodes.filter(n => n.clusterId === cluster.id),
```

Comment:
This needs to be a live array (from `findAll()`) that is then filtered as a computed property in the controller; Any time you do a filter in the route you no longer have a live collection and it will not reflect new or removed nodes as they change while you're still on the page.